docs/audio.rst

```diff
@@ -69,6 +72,13 @@ Functions
 
     Stops all audio playback.
 
+.. py:function:: set_rate(sample_rate)
```

Comment:
Perhaps this could be made into a static-method of the `AudioFrame` class?  That way the scope of it is clear, it only acts on these objects.  And it would then be possible to write:
```py
AudioFrame.set_rate(8000)
...
my_frame = AudioFrame(...)
my_frame.set_rate(8000)
``

Reply:
So, as a static method that would affect all AudioFrame playback instead of individual instances?

I agree that having it as part of `AudioFrame` makes it a lot more clear that it only affects playback of this type, but might be confusing when accessing the function via instances:

```python
frame_one = AudioFrame(...)
frame_two = AudioFrame(...)
frame_one.set_rate(8000)
frame_two.set_rate(8000)
...
while audio.is_playing():
    frame_one.set_rate(scale(accelerometer values...))

# At this point, I'd expect playback for frame_two to still be 8000?
audio.play(frame_two) 
```

Would it'd be more intuitive if each instance hold its playback rate?

Reply:
> So, as a static method that would affect all AudioFrame playback instead of individual instances?

Yes, as a static method calling it once affects all AudioFrame instances.

If we went this way, probably best to document it as `AudioFrame.set_rate(...)` and never refer to it as `frame.set_rate(...)`.  Then it's clear it sets the global rate for all audio frames.

> Would it'd be more intuitive if each instance hold its playback rate?

I'm not sure... maybe?  But then it would be an instance method and you would always call `frame.set_rate(...)` to set the playback rate of that instance.

I think that's possible to implement.